src/components/profile/edit/EditContact.vue

```diff
@@ -21,7 +21,14 @@
           :options="[{ label: 'Primary', value: 'Primary' }]"
           :disabled="true"
         />
-        <input type="text" :value="primaryEmail.value" disabled />
+        <label for="field-email-1-value" class="visually-hidden">Email 1</label>
+        <input
+          type="text"
```

Comment:
`email` I think that makes a keyboard with an @ pop up on mobile.

Reply:
It totally does on my phone. And this will be useful later if we want the browser to validate for us